page/src/components/SelectedEvents.js

```diff
@@ -11,7 +11,11 @@ const SelectedEvents = ({ events, date }) => {
 	<>
 		<h3 className="eventDateDisplay">{formatDate(date)}</h3>
 		<div className="eventsGridDisplay">
-			{events?.map(e => <EventDisplay {...e} />)}
+			{
+				events?.length ? 
+				events.map(e => <EventDisplay {...e} />) : 
+				<p>No event found for that day</p>
```

Comment:
```suggestion
				<p>No events found for this day ☺️.</p>
```

Reply:
You're welcome :) 

Honestly I don't mind changing the text, but for "this" instead of "that", "this day" is usually used as a synonym of "today", which can be confusing, while "that" is rather used to talk about a specific (future or past) day !

Ref : [https://ell.stackexchange.com/a/178569](https://ell.stackexchange.com/a/178569)

Let me know if you want me to change the whole text with your suggestion, or if you prefer to keep "that" as it was 🙂